biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -533,13 +580,26 @@ real(r8) :: tcwood             ! Temperature response function for wood
                      ! the sapwood pools.
                      ! Units are in (kgN/plant)
                      ! ------------------------------------------------------------------
-                     live_stem_n = EDPftvarcon_inst%allom_agb_frac(currentCohort%pft) * currentCohort%bsw / &
-                           woodcn(currentCohort%pft)
-                     live_croot_n = (1.0_r8-EDPftvarcon_inst%allom_agb_frac(currentCohort%pft)) * currentCohort%bsw  / &
-                           woodcn(currentCohort%pft)
-                     froot_n       = currentCohort%br / frootcn(currentCohort%pft) 
-                     
+
+                     select case(hlm_parteh_model)
+                     case (1)
+                        
+                        sapw_c   = currentCohort%prt%GetState(sapw_organ, carbon12_species)
+                        fnrt_c   = currentCohort%prt%GetState(fnrt_organ, carbon12_species)
+
+                        live_stem_n = EDPftvarcon_inst%allom_agb_frac(currentCohort%pft) * &
+                              sapw_c * EDPftvarcon_inst%prt_nitr_stoich_p1(ft,sapw_organ)
+                        
+                        live_croot_n = (1.0_r8-EDPftvarcon_inst%allom_agb_frac(currentCohort%pft)) * &
+                              sapw_c * EDPftvarcon_inst%prt_nitr_stoich_p1(ft,sapw_organ)
+
+                        fnrt_n = fnrt_c * EDPftvarcon_inst%prt_nitr_stoich_p1(ft,fnrt_organ)
                      
+                     case default
+                        
```

Comment:
why does the default case not need to know these things>

Reply:
Good catch. The default should always trigger an error.  And there is no reason I can't code in hypothesis 2 (even though it is not allowed yet).